src/mod_auth_cas.h

```diff
@@ -98,7 +99,7 @@
 #define CAS_DEFAULT_SSO_ENABLED FALSE
 #define CAS_DEFAULT_AUTHORITATIVE FALSE
 
-#define CAS_MAX_RESPONSE_SIZE 65536
+#define CAS_MAX_RESPONSE_SIZE 524288
```

Comment:
Looks like this snuck in and should be changed back to the original.

I'm hoping to git rid of the need to change this in future versions: https://github.com/apereo/mod_auth_cas/pull/132